tox.ini

```diff
@@ -6,13 +6,17 @@ envlist = unit, flake8, integration-sqlserver, integration-synapse
 basepython = python3
 commands = /bin/bash -c '{envpython} -m pytest -v test/integration/sqlserver.dbtspec'
 deps =
-    -r{toxinidir}/requirements.txt
-    -e.
+    -r {toxinidir}/requirements.txt
 
 [testenv:integration-azuresql]
 basepython = python3
 commands = /bin/bash -c '{envpython} -m pytest -v test/integration/azuresql.dbtspec'
 passenv = DBT_AZURESQL_DB DBT_AZURESQL_PWD DBT_AZURESQL_SERVER DBT_AZURESQL_UID
 deps =
-    -r{toxinidir}/requirements.txt
-    -e.
```

Comment:
@swanderz : Why the `-e` editable flag for testing?

Reply:
@JCZuurmond great question! my gut tells me it isn't needed, but I just copied directly from [dbt-spark's `tox.ini`](https://github.com/fishtown-analytics/dbt-spark/blob/90492319adce4345199a59527fc9787228bb25dc/tox.ini#L21-L28). I'm very much a newbie to `pytest` and `tox`. Feel free to remove it!

Reply:
for more, context, [here's me on SO trying to get help](https://stackoverflow.com/questions/65784102/install-local-package-in-tox-unencumbered-by-previous-install-from-github) solving a bizarre bug we were having. The answer was great, but actually gave me even more questions! Maybe there's something useful in there for you..

Reply:
Check, I removed it for both. I am new to `tox`, I have quite some experience with python testing and heard about tox. What I "know" is that it allows you to create different environments to do your testing in. It is useful for things like: run the test suite for python 3.7, 3.8 and 3.9 or a windows and linux system.

I had a quick look at the [tox](https://tox.readthedocs.io/en/latest/index.html) website. Tox uses [virtual environments](https://docs.python.org/3/tutorial/venv.html) to install the package in and then run your test suite. The editable flag is not needed since tox will recreate the environment if it detects a change: `If certain aspects of the project change, a re-creation of the environment is automatically triggered.`

Looking at the tox website I think we use it a bit awkwardly here, but I do not want to change too much in one PR. 

Tox will install your package for your inside the virtual environment it creates, it uses the `setup.py` to do so - `python setup.py sdist` to be precise. However, we tell it to skip this step with the `skipsdist = True` flag. But... we in the end still install the package because there is a dot `.` at the end of the requirements file which tells pip to install the package in the current directory - `pip` is by default used to run the commands in `deps` against.

I think it would make more sense to remove the `skipsdist = True` flag and move the requirements dependencies into the `setup.py`. Seeing that the other `dbt` repos do it in this way, there is a benefit to sticking to it tough.